tests/console/commands/test_new.py

```diff
@@ -57,9 +57,7 @@ def verify_project_directory(
 
     packages = poetry.local_config.get("packages")
 
-    if not packages:
-        assert poetry.local_config.get("name") == package_include.get("include")
```

Comment:
We probably should not remove this test but adapt it. `module_name()`?

Reply:
Agreed. Thanks for the feedback! I have adapted the test.